src/cargo/sources/registry/index.rs

```diff
@@ -226,7 +226,7 @@ pub struct IndexSummary {
 /// * _cache version_ --- Intended to ensure that there's some level of
 ///   future compatibility against changes to this cache format so if different
 ///   versions of Cargo share the same cache they don't get too confused.
-/// * _index format version_ --- The version of the raw index file.
+/// * _index schema version_ --- The schema version of the raw index file.
```

Comment:
My main concern is if this will cause more confusion over when we should bump the version.

However, I can't quite put this into words, so I'm mostly leaving this here in case anyone has additional thoughts, otherwise we can close this out and move on

Reply:
Can definitely drop this. To be honest I just want a way to distinguish “format version” and “cache version”. Maybe the latter should change instead?

Reply:
Just found that this page chooses schema over format. Not sure if they are the same.

https://github.com/rust-lang/cargo/blob/a8ed33fb541739dcbf3da6c7b0600aa41205c410/src/doc/src/reference/registry-index.md?plain=1#L174-L191

Reply:
Looks like it.  Let's go for it then